ValidateOtp/handler.ts

```diff
@@ -87,13 +135,25 @@ export function ValidateOtpHandler(
       .mapLeft<IResponseErrorInternal | IResponseErrorNotFound>(() =>
         ResponseErrorInternal("Cannot validate OTP Code")
```

Comment:
On the errors scenarios, we haven't any log. Could be useful to log for debugging reason. Make sure doesn't log confidential data like text plain OTP.